classes/Customer.php

```diff
@@ -177,7 +176,7 @@ class CustomerCore extends ObjectModel
             'lastname' => array('type' => self::TYPE_STRING, 'validate' => 'isName', 'required' => true, 'size' => 255),
             'firstname' => array('type' => self::TYPE_STRING, 'validate' => 'isName', 'required' => true, 'size' => 255),
             'email' => array('type' => self::TYPE_STRING, 'validate' => 'isEmail', 'required' => true, 'size' => 128),
-            'passwd' => array('type' => self::TYPE_STRING, 'validate' => 'isHashedPassword', 'required' => true, 'size' => 60),
+            'passwd' => array('type' => self::TYPE_STRING, 'validate' => 'isPasswd', 'required' => true, 'size' => 60),
```

Comment:
I'm not sure that is what you need to do to make it work for the BO.

Password can be define by the FO and the BO. Inside the FO, there is a lot of actions to make the new password hashed (and be validate by isHashedPassword. 

What's you need to do it's to encrypt the passwd given inside the BO, I think.